.github/workflows/release.yml

```diff
@@ -13,51 +13,69 @@ on:
   workflow_dispatch:
     inputs:
       version:
-        description: 'release version in format 7.y.z'
+        description: 'The version that is going to be released. Should be in format 7.y.z'
         required: true
+        default: '7.y.z'
   push:
     branches: 
       - 'release'
     paths:
       - 'VERSION'
 
 jobs:
-  release-build-publish:
+  build:
+    name: Create Che Devfile Registry Release
     runs-on: ubuntu-latest
     steps:
-      - name: Clone source code
+      -
+        name: "Checkout  source code"
         uses: actions/checkout@v1
         with:
           fetch-depth: 1
-          ref: release
-
-      - name: Login to Quay.io
-        uses: azure/docker-login@v1
+      - 
+        name: Prepare
+        id: prep
+        run: |
+          SHORT_SHA1=$(git rev-parse --short HEAD)
+          echo ::set-output name=short_sha1::${SHORT_SHA1}
+          VERSION=$(head -n 1 VERSION)
```

Comment:
If this value != `inputs.version` we should use inputs.version.

In fact I'm not sure why we need BOTH a computed value (7.27.0-SNAPSHOT) and an input value (7.27.0). 

I would expect that if the input is missing, we compute it... but I believe part of the release process is to SET the value in the VERSION file, so there's no point reading the file before we set the value. :)  

Instead I'd just take the value from the input as it should be the same as the VERSION file, once the release is complete. 

Correct @mkuznyetsov ?

Reply:
@nickboldt do you expect ci-multiarch workflows should be formatted https://github.com/eclipse/che-dashboard/blob/master/.github/workflows/ci-multiarch.yaml#L91-L115 referenced to https://github.com/eclipse/che-dashboard/pull/119#issuecomment-783374087

Reply:
> Instead I'd just take the value from the input as it should be the same as the VERSION file, once the release is complete.

@nickboldt can we modify the version file `7.27.0-SNAPSHOT->7.27.0` apply the condition if `[vesion==input.version]` then release the specific version

Reply:
@nickboldt as of now, make-release.sh is called first from another workflow, which makes changes to VERSION file, and then this workflow does the rest. Here it reads the VERSION file, in case it was triggered by the push to the release branch, which has no inputs

Because it was slightly confusing, I made a PR (about which I've forgot 😃 )
https://github.com/eclipse/che-devfile-registry/pull/338/files
it makes `make-release.sh` will be called from this workflow

Reply:
So which approach are we taking? Right now as it stands, the `7.27.0-SNAPSHOT` problem won't happen because the workflow is triggered after the file is changed. But we can have this PR wait until https://github.com/eclipse/che-devfile-registry/pull/338/files is merged. What do you guys think? cc @nickboldt @mkuznyetsov

Reply:
PR 338 is ready to merge, after fixing all the shellcheck complaints. https://github.com/eclipse/che-devfile-registry/pull/338/files

Reply:
I've merged it, @bivasda1 please adjust the PR to make sure `-SNAPSHOT` isn't pushed as part of the tag